skimage/draw/draw.py

```diff
@@ -107,22 +108,33 @@ def ellipse(r, c, r_radius, c_radius, shape=None, rotation=0.):
            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
            [1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 1, 1]], dtype=uint8)
     """
+    # <-- Starting deprecation warnings: rotation -> orientation,
+    # shape <-> orientation.
+    if rotation is not None:
+        warn('The keyword rotation will be replaced with orientation in version '
+             '0.19.', DeprecationWarning, stacklevel=2)
```

Comment:
Deprecation warnings are hard to spot. We have been leaving them as standard warnings. I think deprecation warnings are better for those longer deprecation cycles